KokoDajMu/KokoDajMu_MAIN/Classes/WriteInFile.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Text.RegularExpressions;
+using System.Linq;
+
+namespace KokoDajMu.Classes
+{
+    public class WriteInFile
+    {
+        private static ReadFromFile reader = new ReadFromFile();
+        private List<User> usersList = reader.GetUsersFromFile();
+        private List<Listener> listenersList = reader.GetListenersFromFile();
+        private List<Artist> artistsList = reader.GetArtistFromFile();
+        private List<Album> albumsList = reader.GetAlbumFromFile();
+        private List<Song> songsList = reader.GetSongFromFile();
+        private User testUser = new User("Koko", "Koko2311", "Listener");
+        private Listener testListener = new Listener();
+        private Artist testArtist = new Artist();
+        private Album testAlbum = new Album("KokoAlbum2021");
+        private Song testSong = new Song("KokoMix2021", "36000");
+
+        public void SaveUserListInTextFile()
+        {
+            TextWriter convertUserListInTextFile = new StreamWriter(@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\UserList.txt");
```

Comment:
Can be extracted in a more global scope rather than the method scope , as it is used in multiple methods.